x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/translations.ts

```diff
@@ -35,6 +35,14 @@ export const UPLOAD_VALUE_LISTS_TOOLTIP = i18n.translate(
   }
 );
 
+export const UPLOAD_VALUE_LISTS_PRIVILEGES_TOOLTIP = i18n.translate(
+  'xpack.securitySolution.lists.detectionEngine.rules.uploadValueListsButtonPrivilegesTooltip',
+  {
+    defaultMessage:
+      'A user with manage cluster privileges must visit the Rules page before you can import value lists',
```

Comment:
Can you style `manage` so it stands out a bit more? If not, no worries, figured I'd ask.

```suggestion
      'A user with manage cluster privileges must visit the Rules page before you can import value lists.',
```